src/app/shared/components/template/components/task-card/task-card.component.ts

```diff
@@ -51,8 +119,14 @@ export class TmplTaskCardComponent extends TemplateBaseComponent implements OnIn
     this.title = getStringParamFromTemplateRow(this._row, "title", null);
     this.subtitle = getStringParamFromTemplateRow(this._row, "subtitle", null);
     this.image = getStringParamFromTemplateRow(this._row, "image", null);
+
+    // Use "variant" param to determine styles, including support for legacy "style" param
     this.style = getStringParamFromTemplateRow(this._row, "style", "landscape");
-    this.isButton = this.style.includes("button");
+    this.variant = getStringParamFromTemplateRow(this._row, "variant", "landscape")
+      .replace(",", " ")
+      .replace("  ", " ");
```

Comment:
If we don't want to support authors supplying a comma-separated of variants, we could remove these, otherwise if we expand this `variant` feature across components it could be worth wrapping them inside a `getStringParamFromTemplateRow()` util function

Reply:
That script will only replace the first instance of a comma, you'll likely need regex e.g. `.replace(/,/g,' ')`, or maybe more simple to just use `.split(',').join(' ')`